encord/objects/internal_helpers.py

```diff
@@ -390,12 +388,11 @@ def __repr__(self):
 
 
 def get_default_answer_from_attribute(attribute: Attribute) -> Answer:
-    property_type = attribute.get_property_type()
```

Comment:
This was better before?

Reply:
Apparently that is the pythonic way. Blow my mind coming from Java :D

Reply:
@adam-encord @rad-cord the whole point is that this way mypy can do type narrowing when necessary.

Reply:
https://mypy.readthedocs.io/en/stable/type_narrowing.html